czi_tables/module_analysis_mapping.txt

```diff
@@ -5,6 +5,8 @@
 # Trajectories (T)
 # Cell type alignment (CT)
 # Dimensionality reduction (DR)
+scanpy_filter_cells:QC
+scanpy_filter_genes:QC
 scanpy_find_cluster:C
 scanpy_find_markers:DE-MD
 scanpy_find_variable_genes:P
```

Comment:
Maybe @a-solovyev12 could add some entries here for the cell typing tools? Or you could ask Ruben as well.